README.md

```diff
@@ -6,17 +6,14 @@
 [![Docker Pulls][docker-badge]][docker]
 [![CII Best Practices][cii-badge]][cii]
 
-:balloon: Welcome to Linkerd! :wave:
+** This repo is for the old, 1.x version of Linkerd. Linkerd development is now
+happening in the [linkerd2][l5d2] repo.**
 
-Linkerd is a transparent *service mesh*, designed to make modern applications
-safe and sane by transparently adding service discovery, load balancing, failure
-handling, instrumentation, and routing to all inter-service communication.
-
-Linkerd (pronounced "linker-DEE") acts as a transparent HTTP/gRPC/thrift/etc
-proxy, and can usually be dropped into existing applications with a minimum of
-configuration, regardless of what language they're written in. It works with
-many common protocols and service discovery backends, including scheduled
-environments like Mesos and Kubernetes.
+Linkerd 1.x (pronounced "linker-DEE") acts as a transparent
+HTTP/gRPC/thrift/etc proxy, and can usually be dropped into existing
+applications with a minimum of configuration, regardless of what language
+they're written in. It works with many common protocols and service discovery
+backends, including scheduled environments like Mesos and Kubernetes.
 
```

Comment:
```suggestion

Its fully featured with powerful load balancing algorithms, failure handling, instrumentation, 
distributed tracing and dynamic request routing.

It also offers a stand-alone version of the service discovery components called Namerd, 
which enables a scalable architecture with centralised namers and a simple api for adaptation 
to other load balancing systems. 

```

Reply:
This is a nice description, just a bit of punctuation that is missing:

```
Linkerd is fully featured with powerful load balancing algorithms, failure handling, instrumentation, 
distributed tracing, and dynamic request routing.
It also offers a stand-alone version of the service discovery components, called Namerd, 
which enables a scalable architecture with centralised "namers" and a simple api for adaptation 
to other load balancing systems. 

```